sounds/management/commands/analysis_orchestrator.py

```diff
@@ -0,0 +1,96 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+import json
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from sounds.models import Sound, SoundAnalysis
+
+console_logger = logging.getLogger("console")
+
+# Dictionary that links simple names and versions of the analyzers to their actual docker images
```

Comment:
I guess this is what should be defined in the configuration file? Why do we want this mapping? we could probably simply use the name of the docker images

Reply:
I thought we wanted to have simplified names and versions, so they were easily comparable – the versions specifically, as integers. I don't know why we wanted this for, though...

Reply:
yeah, we want simple names. I think we should modify the makefile for the analyzers build process to simply use the folder name as image name. then it would be nice to add a label to the docker image which includes the current date and commit (see example here: https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d, using LABEL and arguments should be easy). I found however that it is good that the docker image name is not changing all the time.

if the "version" is not in the folder name, then we're somehow loosing versioning in the docker image name. maybe we could re-add it by somehow asking analyzer developers to specify "version" in a file inside the folder (eg, place a file named "VERSION" with a tag like "v1"), and then from the makefile read that version file and add it to the image name (so we get analyzer_name:version). I think that'd work well.